.gitignore

```diff
@@ -35,6 +35,9 @@ cmd/monaco/test-resources/download/
 .env.production.local
 config.json
 
+yarn.lock
```

Comment:
Please do not ignore that file. This locks the dependencies and thereby ensures
that if anybody wants to build it locally, they get the same versions as the build 
would have.

Reply:
@patrickpichler noted